palantir-java-format/src/main/java/com/palantir/javaformat/OpsBuilder.java

```diff
@@ -303,6 +306,12 @@ public void close() {
         return peekToken(0);
     }
 
+    /** Return whether the last token emitted is followed by a newline. */
+    public boolean lastTokenFollowedByNewline() {
```

Comment:
s/lastToken/mostRecentToken/ ? I found the `last` thing a tiny bit ambiguous (like is this referring to the very last one in a long list somehow)